src/collar/oc_rlvextension.lsl

```diff
@@ -286,30 +282,32 @@ MenuSetValue(key kID, integer iAuth, string sValueName) {
 integer g_iLastOwnerEx;
 integer g_iLastTrustedEx;
 
-updateList(list newlist, integer type) // type 1=owner, 2=trusted;
+integer LIST_TYPE_OWNER = 1;
```

Comment:
Naked magic numbers passed into functions can be a big impediment to readability.  We should use constants unless the script is actually crashing from running out of memory, and even then, this would be one of the last things to attempt to get some memory back.